### PR TITLE
init project structure

### PR DESCRIPTION
**Pull request by abdullahalamodi**
don't delete template files until we replace them with real flies

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
     <application
         android:allowBackup="true"
+        android:name=".data.ApplicationClass"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".view.MainActivity">
```
Comment (MaimoonaAlrozmi) [inline app/src/main/AndroidManifest.xml:13]: why did u use "view"

**Comment by MaimoonaAlrozmi** [comment]
init project structure "Merge with abdullah alamodi"

**Review by abdullahalamodi — COMMENTED** [review]
reviewed

**Comment by abdullahalamodi** [comment]
mistake

**Pull request closed without merging**